### ISSUE TITLE
step tabular static fields

### ISSUE THREAD

**Issue by paskuale75** [buttflattery/yii2-formwizard#91]
**Is your feature request related to a problem? Please describe.**
I've static elements, I don't need the add and remove functionality is there an option to disable them?
**Describe the solution you'd like**
remove add/remove functions

**Describe alternatives you've considered**

**Additional context**

**Comment by paskuale75** [comment]
...this project is still alive ?

**Comment by buttflattery** [comment]
yes but unfortunately I am not getting enough time for this project at the moment @paskuale75 will follow up on this after looking into the code section as I dont remember exactly at the moment. Hopefully on this weekend i will respond

**Comment by paskuale75** [comment]
Ah ok, good because this extension seems to be the only one out there to take advantage of a multi-step form. Thanks for reply.
